shell/detail/provisioning.cattle.io.cluster.vue

```diff
@@ -435,7 +248,40 @@ export default {
       ];
     },
 
-    mgmtNodeSchemaHeaders() {
+    machinePaginatedHeaders() {
+      return [
+        STEVE_STATE_COL,
+        STEVE_NAME_COL,
+        {
+          name:          'node-name',
+          labelKey:      'tableHeaders.machineNodeName',
+          sort:          false,
+          search:        false,
+          value:         'status.nodeRef.name',
+          formatter:     'LinkDetail',
+          formatterOpts: { reference: 'kubeNodeDetailLocation' },
+          dashIfEmpty:   true,
+        },
+        {
+          ...INTERNAL_EXTERNAL_IP,
+          sort:   false,
+          search: false
+        },
+        {
+          ...MACHINE_NODE_OS,
```

Comment:
we might be able to sort on this, though it would be a bit off with the empty value (which defaults to linux)

Reply:
(same for mgmt nodes)